deployments/ecs/ec2/README.md

```diff
@@ -0,0 +1,237 @@
+# Amazon ECS EC2 Deployment
+Familiarity with Amazon ECS using launch type EC2 is assumed. Consult the 
+[Getting started with the Amazon ECS console using Amazon EC2](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/getting-started-ecs-ec2.html)
+for further reading.
+
+Unless stated otherwise, the
+[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed as a **sidecar** (additional container) to ECS tasks.
```

Comment:
Why it has to be sidecar for ECS on EC2? Can't we deploy it as a separate task in Daemon mode? Similar to [signalfx-agent instructions](https://github.com/signalfx/signalfx-agent/tree/main/deployments/ecs). I think it'll be much easier for user and will give them host metrics out of the box.

Reply:
I changed the instructions as you suggested. I still have to add host metrics to the default config.